[Bug] zos_encode failing because of missing function uss_tag_encoding

### Is there an existing issue for this?

- [X] There are no existing issues.

### Are the dependencies a supported version?

- [X] The dependencies are supported.

### IBM Z Open Automation Utilities

v1.2.2

### IBM Enterprise Python

v3.9.x

### IBM z/OS Ansible core Version

v1.5.0

### ansible-version

latest

### z/OS version

None

### Ansible module

_No response_

### Bug description

Function uss_tag_encoding was added on https://github.com/ansible-collections/ibm_zos_core/commit/5ba30a8badd95fe13f6f12e19371c4f849f8d2fa but somehow is not there anymore, this is breaking the test cases for zos_encode.

### Test suite output 
```
functional/modules/test_zos_encode_func.py ..F

=================================== FAILURES ===================================
__________________ test_uss_encoding_conversion_without_dest ___________________

ansible_zos_module = <pytest_ansible.host_manager.v28.HostManagerV28 object at 0x7f50db246eb0>

    def test_uss_encoding_conversion_without_dest(ansible_zos_module):
        try:
            hosts = ansible_zos_module
            hosts.all.copy(content=TEST_DATA, dest=USS_FILE)
            results = hosts.all.zos_encode(
                src=USS_FILE,
                encoding={
                    "from": FROM_ENCODING,
                    "to": TO_ENCODING,
                },
            )
            pprint(vars(results))
            for result in results.contacted.values():
                assert result.get("src") == USS_FILE
                assert result.get("dest") == USS_FILE
                assert result.get("backup_name") is None
>               assert result.get("changed") is True
E               AssertionError: assert False is True
E                +  where False = <built-in method get of dict object at 0x7f50dadbb300>('changed')
E                +    where <built-in method get of dict object at 0x7f50dadbb300> = {'_ansible_no_log': False, 'changed': False, 'dest': '/tmp/encode.data', 'failed': True, ...}.get

functional/modules/test_zos_encode_func.py:129: AssertionError
----------------------------- Captured stdout call -----------------------------
{'contacted': {'ec33012a.vmec.svl.ibm.com': {'_ansible_no_log': False,
                                             'changed': False,
                                             'dest': '/tmp/encode.data',
                                             'failed': True,
                                             'gid': 1,
                                             'group': 'OMVSGRP',
                                             'invocation': {'module_args': {'backup': False,
                                                                            'backup_compress': False,
                                                                            'backup_name': None,
                                                                            'dest': None,
                                                                            'encoding': {'from': 'IBM-1047',
                                                                                         'to': 'ISO8859-1'},
                                                                            'from_encoding': 'IBM-1047',
                                                                            'src': '/tmp/encode.data',
                                                                            'tmp_hlq': None,
                                                                            'to_encoding': 'ISO8859-1'}},
                                             'mode': '0644',
                                             'msg': 'AttributeError("\'EncodeUtils\' '
                                                    'object has no attribute '
                                                    '\'uss_tag_encoding\'")',
                                             'owner': 'OBAGAOI',
                                             'size': 480,
                                             'src': '/tmp/encode.data',
                                             'state': 'file',
                                             'uid': 0}}}
```

### Playbook verbosity output.

_No response_

### Ansible configuration.

_No response_

### Contents of the inventory

_No response_

### Contents of `group_vars` or `host_vars`

_No response_

--- a/plugins/module_utils/encode.py
+++ b/plugins/module_utils/encode.py
@@ -454,21 +454,21 @@ class EncodeUtils(object):
         return convert_rc
 
     def uss_tag_encoding(self, file_path, tag):
-         """Tag the file/directory specified with the given code set.
-         If `file_path` is a directory, all of the files and subdirectories will
-         be tagged recursively.
-         Arguments:
-             file_path {str} -- Absolute file path to tag.
-             tag {str} -- Code set to tag the file/directory.
-         Raises:
-             TaggingError: When the chtag command fails.
-         """
-         is_dir = os.path.isdir(file_path)
+        """Tag the file/directory specified with the given code set.
+        If `file_path` is a directory, all of the files and subdirectories will
+        be tagged recursively.
+        Arguments:
+            file_path {str} -- Absolute file path to tag.
+            tag {str} -- Code set to tag the file/directory.
+        Raises:
+            TaggingError: When the chtag command fails.
+        """
+        is_dir = os.path.isdir(file_path)
 
-         tag_cmd = "chtag -{0}c {1} {2}".format("R" if is_dir else "t", tag, file_path)
-         rc, out, err = self.module.run_command(tag_cmd)
-         if rc != 0:
-             raise TaggingError(file_path, tag, rc, out, err)
+        tag_cmd = "chtag -{0}c {1} {2}".format("R" if is_dir else "t", tag, file_path)
+        rc, out, err = self.module.run_command(tag_cmd)
+        if rc != 0:
+            raise TaggingError(file_path, tag, rc, out, err)
 
     def uss_file_tag(self, file_path):
         """Returns the current tag set for a file.

--- a/plugins/module_utils/encode.py
+++ b/plugins/module_utils/encode.py
@@ -453,6 +453,23 @@ class EncodeUtils(object):
 
         return convert_rc
 
+    def uss_tag_encoding(self, file_path, tag):
+         """Tag the file/directory specified with the given code set.
+         If `file_path` is a directory, all of the files and subdirectories will
+         be tagged recursively.
+         Arguments:
+             file_path {str} -- Absolute file path to tag.
+             tag {str} -- Code set to tag the file/directory.
+         Raises:
+             TaggingError: When the chtag command fails.
+         """
+         is_dir = os.path.isdir(file_path)
+
+         tag_cmd = "chtag -{0}c {1} {2}".format("R" if is_dir else "t", tag, file_path)
+         rc, out, err = self.module.run_command(tag_cmd)
+         if rc != 0:
+             raise TaggingError(file_path, tag, rc, out, err)
+
     def uss_file_tag(self, file_path):
         """Returns the current tag set for a file.
         Arguments:

